extension/ts/extensions/warning.ts

```diff
@@ -22,13 +24,14 @@ const addUndoEditListener = () =>
 		document.getElementById("book_editTabTextDelete"), // so that it doesn't alert you when you're deleting something (?)
 	].forEach((element) => element?.addEventListener("click", undoEdits));
 
-window.addEventListener(FORM_RENDER_EVENT, () => {
-	addEditListener();
+onFormRender((ignored, forEachElement: ForEachFormElement) => {
```

Comment:
nit: `ignored` -> `_ignored` so that vs code stops crying

![Screen Shot 2022-10-06 at 4 07 09 PM](https://user-images.githubusercontent.com/63937200/194434414-ccdc082e-aaa3-4281-98c4-5b9dd568d08e.png)

Reply:
there must be a setting to make `ignored` actually ignored.

does `_ignored` actually work????

Reply:
ik you fixed it but it does lmao :skull: 

![Screen Shot 2022-10-06 at 4 49 05 PM](https://user-images.githubusercontent.com/63937200/194438371-0b0248e6-5a12-4162-831a-bdfaa64426e1.png)